### PR TITLE
Make breadth explicit; handle Dodge Block Parry.

### PR DESCRIPTION
**Pull request by johnlamping**

### CODE REVIEW

**Comment by johnlamping** [comment]
This brings the sheet up to date with two rules clean-ups that were enabled by the new cost structure: 
   There are one again 3 different defense skills: Dodge, Block, Parry. (Before, if you were a Blocker, your ability to Dodge would be too low if you got attacked from behind.)
   You now get one Breadth skill per discipline. (And you mark them explicitly on the sheet. It doesn't check for one per discipline)

File: ui/scripts/inlineHtml/inlineScripts.js [hunk 1]
```diff
@@ -1021,6 +990,7 @@ const updateSkillDerivedForId = function (section, row_id) {
             Number(values[base]) +
             disciplineexpertise_n +
             (expertise_v === 'ST' ? -1 :
+              expertise_v === 'B' ? -1 :
               expertise_v !== '--' ? Number(expertise_v) :
```
Comment (jazeee) [inline ui/scripts/inlineHtml/inlineScripts.js]: Generally, nested ternaries are confusing. As a follow up, I think we should split this up to increase clarity.
Perhaps a clean up follow up.

**Review by jazeee — APPROVED** [review]
Looks good
🚀 🚢

File: ui/scripts/inlineHtml/inlineScripts.js (hunk 1 above)
Comment (johnlamping) [reply ui/scripts/inlineHtml/inlineScripts.js]: I think of nested ternaries that are laid out like this as the equivalent of case statements, which I'm familiar with, and so don't find confusing, personally.
What would a split up version look like?

Comment (jazeee) [reply ui/scripts/inlineHtml/inlineScripts.js]: I'd have to look at it, but I'd likely extract named variables. Often, this tends to give the code more context and readability as well.
That being said, this is probably fine for the code base.

**Comment by johnlamping** [comment]
Is this an improvement on the ability computation?

File: ui/scripts/inlineHtml/inlineScripts.js (hunk 1 above)
Comment (johnlamping) [reply ui/scripts/inlineHtml/inlineScripts.js]: Is this an improvement on the ability computation?

Comment (jazeee) [reply ui/scripts/inlineHtml/inlineScripts.js]: I think nested ternaries will always be a problem, and are generally linted against.
https://eslint.org/docs/latest/rules/no-nested-ternary

Code formatters will auto-format things to look worse or better, so we can't rely on formatting to keep things managed.

The typical directions that I tend to go towards is a helper function with either switch case or if returns.
This also helps isolate code and improves encapsulation for testing purposes.

That all being said, this isn't code for production (i.e. not work 😸 )

**Pull request merged**